public/src/app.js

```diff
@@ -35,18 +37,18 @@ const app = (s) => {
     socket.on("connected", (socketID) => {
       LocalStorage.set("pwf_socket", socketID);
       initUsername(socketID);
+    });
 
-      socket.on("update", (paintProperties) => {
-        updateDrawing(s, paintProperties);
-      });
+    socket.on("update", (paintProperties) => {
```

Comment:
In an earlier attempt to deal with this issue I tried nesting all these WS listeners inside the main `connected` one, but I don't think it makes a difference so I'm just reverting that change here.